[WIP] Payment account support for the API

blabno opened the pull request:
The only non-experimental endpoint is to get list of payment accounts so that user can retrieve account's id to use during trade.
The rest of endpoints (create, remove payment account) are marked as experimental features and are required for integration tests.

ManfredKarrer committed:
Add onInitP2pNetwork and onInitWallet to BisqSetupListener

- Rename BisqSetupCompleteListener to BisqSetupListener
- Add onInitP2pNetwork and onInitWallet to BisqSetupListener
- make onInitP2pNetwork and onInitWallet default so no impl. required

ManfredKarrer committed:
Start server at onInitWallet and add wallet password handler

- Add onInitWallet to HttpApiMain and start http server there
- Add onRequestWalletPassword to BisqSetupListener
- Override setupHandlers in HttpApiHeadlessApp and adjust
setRequestWalletPasswordHandler (impl. missing)
- Add onRequestWalletPassword to HttpApiMain

ManfredKarrer committed:
Add periodic log to remind user to enter pw

freimair committed:
Init BsqBlocks metric

oscarguindzberg committed:
Use bitcoinj with connection handling audit fixes + Transaction.unCache() set confidence to null

ManfredKarrer committed:
Remove referralIdInputTextField

The referralId support did not get any interest so we removed it from
the UI. It is still supported via prog argument and can be re-enabled
again once there are use cases (API?).
I left it commented out to make re-enabling easier.

ManfredKarrer committed:
Ignore utxo below a user defined threshold to avoid dust attacks

Implements https://github.com/bisq-network/bisq/issues/2604

ripcurlx committed:
Add proposal as shared translation

ripcurlx committed:
Add votes as shared translation

m4hdi1995 committed:
List ParsiCoin (PARS)

Commit:
Update CurrencyUtil.java

add BURST+STEEM to the list of removed assets

ManfredKarrer committed:
Fix wrong order or arguments

ManfredKarrer committed:
Make base class protected and remove Inject annotation

ManfredKarrer committed:
Merge pull request #2674 from ManfredKarrer/add-more-info-to-dao-monitor

Handle dao state conflicts better

ripcurlx committed:
Add DAO launch popup

ripcurlx committed:
Adapt update popup to general information popup and darken blurred background for regular popups

ripcurlx committed:
Remove unnecessary negative left padding

ripcurlx committed:
Make blurred background even darker based on #2337.

ManfredKarrer committed:
Merge pull request #2690 from ripcurlx/align-hyperlinks

Remove unnecessary negative left padding

ripcurlx committed:
Updated translations based on https://github.com/bisq-network/bisq/pull/2688#issuecomment-481709119

ripcurlx committed:
Darken background less, to feel less dark.

ManfredKarrer committed:
Merge branch 'master' into rc_v1.0.0

# Conflicts:
#	common/src/main/proto/pb.proto
#	core/src/main/java/bisq/core/dao/governance/blindvote/storage/BlindVotePayload.java

ripcurlx committed:
Improve cheaper fees screenshot and popup layout

ripcurlx committed:
Change dao link

ManfredKarrer committed:
Merge pull request #2689 from ripcurlx/adapt-update-popup

Adapt update popup

ManfredKarrer committed:
Merge pull request #2688 from ripcurlx/add-dao-launch-popup

Add DAO launch popup

ManfredKarrer committed:
Fix resource key

ManfredKarrer committed:
Add TODO for fixing issue with not disabled buttons

ripcurlx committed:
Add slide animation

ripcurlx committed:
Merge pull request #2680 from HarryMacfinned/update-removed-assets-list

Update CurrencyUtil.java

ripcurlx committed:
Remove non-core languages based on discussion on Transifex

ripcurlx committed:
Save horizontal space and clean-up MainView

ripcurlx committed:
Update transifex script with new core languages

ripcurlx committed:
Move DontShowAgainCheckbox into same HBox as buttons to prevent overlapping. Fixes #2681.

ripcurlx committed:
Set close button to same width as action button to not truncate translations in that case

ripcurlx committed:
Add missing import

ripcurlx committed:
Add missing check to prevent re-selection of selected select-proposal-window that was just closed

ManfredKarrer committed:
Merge pull request #2700 from ripcurlx/fix-deselection-of-proposal

Add missing check to prevent re-selection of proposal

ManfredKarrer committed:
Merge pull request #2699 from ripcurlx/improve-overlay-button-handling

Improve space management in button bar of overlays

ManfredKarrer committed:
Merge pull request #2696 from ripcurlx/add-slide-animation

Add slide animation for DAO launch popup

ManfredKarrer committed:
Merge pull request #2697 from ripcurlx/remove-non-core-languages

Remove non core languages

ManfredKarrer committed:
Merge branch 'master' into rc_v1.0.0

ManfredKarrer committed:
Fix rowSpan

ManfredKarrer committed:
Fix issue with animation at start

ManfredKarrer committed:
Add isHiddenProperty and displayOrderPriority

Needed to support a priority queue for adding multiple popups and then
execute the display ordered by priority.

ManfredKarrer committed:
Use priority queue for handling multiple popups

ManfredKarrer committed:
Remove random string for dev testing

ManfredKarrer committed:
Use betanet genesis for mainnet

ManfredKarrer committed:
Use new dummy genesis tx for mainnet

ripcurlx committed:
Merge pull request #2701 from ManfredKarrer/manage-startup-popups

Manage startup popups

ripcurlx committed:
Add slide out animation and delay initial slide in animation to increase the chance that it is seen (dependent on available processing power on startup)

ripcurlx committed:
Bump version number

ManfredKarrer committed:
Merge pull request #2703 from ripcurlx/improve-slide-behavior

Add slide out animation

ManfredKarrer committed:
Reduce initial delay for animation

devinbileck committed:
Add specific URL validation for proposal links

Since all proposals should be created in GitHub, and we have URL
redirects in place in case we switch platforms in the future, enforce
proposal links to utilize the specific URL redirect.

Resolves #2473

ManfredKarrer committed:
Merge branch 'master' into rc_v1.0.0

ManfredKarrer committed:
Merge pull request #4 from ripcurlx/bump-version-number

Bump version number

ManfredKarrer committed:
Improve filter

devinbileck committed:
Update links based on https://github.com/bisq-network/bisq-website/pull/164

ripcurlx committed:
Reduce min width of close button to prevent truncations

ripcurlx committed:
Fix wrong padding of links in trade feedback popup

ManfredKarrer committed:
Use full address instead of hostname

ManfredKarrer committed:
Merge pull request #2707 from ManfredKarrer/fix-filter

Use full address instead of hostname

ManfredKarrer committed:
Merge branch 'master' into rc_v1.0.0

ManfredKarrer committed:
Change most payment methods from  DEFAULT_TRADE_LIMIT_MID_RISK to DEFAULT_TRADE_LIMIT_HIGH_RISK

ManfredKarrer committed:
Reduce sec deposits

ManfredKarrer committed:
Add requireUpdateToNewVersion check

ManfredKarrer committed:
Make check case insensitive

ripcurlx committed:
Move country into second column to save space

ripcurlx committed:
Set new minimum deposit if deposit lower than minimum was persisted

ManfredKarrer committed:
Merge pull request #5 from ripcurlx/improve-sepa-form

Small improvements

ManfredKarrer committed:
Fix nullpointer

ManfredKarrer committed:
Add missing parameter in test

ripcurlx committed:
Update translations

ripcurlx committed:
Update Mainnet dbs and remove old test dbs

ripcurlx committed:
Reduce table height and update height calculation to prevent scrolling

ripcurlx committed:
Align Altcoin list with Altcoin selection combobox

ManfredKarrer committed:
Merge pull request #7 from ripcurlx/align-currency-list-with-combobox

Align Altcoin list with Altcoin selection combobox

ManfredKarrer committed:
Merge pull request #6 from ripcurlx/fix-scrolling-in-trade-view

Reduce table height and update height calculation to prevent scrolling

ManfredKarrer committed:
Merge branch 'master' into rc_v1.0.0

ManfredKarrer committed:
Set mainnet Genesis Tx ID

ripcurlx committed:
Revert to SNAPSHOT

ManfredKarrer committed:
Rebroadcast new blocks at lite nodes

ManfredKarrer committed:
Change log levels

ManfredKarrer committed:
Change log levels

ManfredKarrer committed:
Merge branch 'master' into forward-broadast-msg

devinbileck committed:
Add missing pricenode project

When generating shell scripts in the root project directory, the
pricenode project was missing.

ripcurlx committed:
Merge pull request #2722 from devinbileck/update-build.gradle

Add missing pricenode project generated shell script

ripcurlx committed:
Fix wrong calculation of percentage based security deposit

ripcurlx committed:
Show DAO news badge to increase visibility of DAO launch

ripcurlx committed:
Only show rounding popup once during activation and not show it in edit offer screen.
Fixes #2708.

ripcurlx committed:
Merge pull request #2723 from ripcurlx/fix-calculation-of-sec-deposit-in-edit-offer

Fix wrong calculation of percentage based security deposit

ManfredKarrer committed:
Merge pull request #2725 from ripcurlx/remove-rounding-popup-showing-redundancy

Only show rounding popup once during activation

ripcurlx committed:
Merge pull request #2724 from ripcurlx/show-news-badge-for-dao-launch

Show DAO news badge to increase visibility of DAO launch

ripcurlx committed:
Merge pull request #2719 from ManfredKarrer/forward-broadast-msg

Rebroadcast new blocks at lite nodes

ripcurlx committed:
Bump version number

ripcurlx committed:
Revert to SNAPSHOT version

ManfredKarrer committed:
Apply formatting (no code change)

ManfredKarrer committed:
Add TODO

ManfredKarrer committed:
Add size method

ManfredKarrer committed:
Only apply FULL_DAO_NODE if arg value is "true"

ManfredKarrer committed:
Avoid logging calls from monitor nodes

Monitor nodes have only one capability set. We don't want to get tons
of logs from those (trade statistics, account witness,....).

oscarguindzberg committed:
Doichain: set p2shHeader.

See https://github.com/Doichain/core/blob/master/src/chainparams.cpp#L181

ManfredKarrer committed:
Always use AppSetupWithP2PAndDAO independent of isDaoFullNode

ManfredKarrer committed:
Use AppSetupWithP2PAndDAO instead iof AppSetupWithP2P

ManfredKarrer committed:
Sync up code with SeedNodeMain

ManfredKarrer committed:
Merge pull request #2728 from oscarguindzberg/fix-doichainparams

Doichain: set p2shHeader.

ManfredKarrer committed:
Request blocks in case we have not received it

Once we get a new block from BitcoinJ we start a timer and check in 10
seconds if we have received the block from the P2P network. If not we
request it from the seed nodes.

ManfredKarrer committed:
Add check for nullable senderNodeAddress

ManfredKarrer committed:
Only set capabilities if not null

ManfredKarrer committed:
Make GetBlocksRequest implement SendersNodeAddressMessage and SupportedCapabilitiesMessage

When we request at startup the blocks from a seed which has not yet
received out capabilities or node address we will not get a response.
We do not want to depend on the state in the getData domain in the
p2p network by using only a seed which has already responded but we
prefer to make sure the seednode will get all the data by our request
to be able to respond.

ManfredKarrer committed:
Use fullAddress for ignored peers and in peer info

ManfredKarrer committed:
Use full address instead of hostname

Mixing usage of host name and full name caused a few bugs in the past.
We should use only full address by default.

ripcurlx committed:
Remove BSQ filter for crypto currencies

ripcurlx committed:
Merge pull request #2733 from ManfredKarrer/fix-ignore-list

Fix ignore list

ManfredKarrer committed:
Merge pull request #2732 from ManfredKarrer/add-nodeaddress-at-getblocks-request

Add nodeaddress and capabilities at getblocks request

ManfredKarrer committed:
Merge pull request #2734 from ripcurlx/include-bsq-in-markets-print-tool

Remove BSQ filter for crypto currencies

Commit:
List ZeroClassic (ZERC)

ManfredKarrer committed:
Fix issues with DAO full mode in preferences

There was missing the block notify port as well as the options have
not been applied correctly which led to the situation that when you
run the same application in full mode and later in lite mode that
data got persisted.

Added also to the footer a note if full mode is running.

ManfredKarrer committed:
Merge pull request #2621 from ManfredKarrer/ignore-dust-utxo

Ignore dust utxo

ManfredKarrer committed:
Request 1 block above our chain height

ManfredKarrer committed:
Fix missing argument in test class

ManfredKarrer committed:
Fix missing argument in test class

ManfredKarrer committed:
Merge pull request #2730 from ManfredKarrer/request-block-if-not-received

Request blocks in case we have not received it

ManfredKarrer committed:
Add UTF8 for getBytes calls

Fixes https://github.com/bisq-network/bisq/issues/2729

ripcurlx committed:
Improve handling of editing of offers with invalid security deposit

ripcurlx committed:
Use different security deposit for fiat-crypto and crypto-crypto trades

ripcurlx committed:
Use existing styling for developer alert message

ripcurlx committed:
Remove unnecessary static import

ManfredKarrer committed:
Show readable data in offer

ManfredKarrer committed:
Add filter box to trade history

Can filter by onion address, date string, trade id and
payment details data.

ManfredKarrer committed:
Merge pull request #2740 from ripcurlx/handle-editing-of-offers-with-invalid-deposit

Improve handling of editing of offers with invalid security deposit

ManfredKarrer committed:
Merge pull request #2742 from ripcurlx/set-lower-security-deposit-for-altcoins

Use different security deposit for fiat-crypto and crypto-crypto trades

ManfredKarrer committed:
Merge pull request #2746 from ripcurlx/fix-styling-of-alert-message

Use existing styling for developer alert message

ManfredKarrer committed:
Merge branch 'master' of https://github.com/bisq-network/bisq

ManfredKarrer committed:
Merge branch 'master' into fix-dao-full-mode-settings-issue

# Conflicts:
#	common/src/main/proto/pb.proto
#	core/src/main/java/bisq/core/user/Preferences.java
#	core/src/main/java/bisq/core/user/PreferencesPayload.java
#	desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java

ManfredKarrer committed:
Fix missing arguments

ManfredKarrer committed:
Merge branch 'fix-dao-full-mode-settings-issue' of https://github.com/ManfredKarrer/bisq into fix-dao-full-mode-settings-issue

ManfredKarrer committed:
Fix account age display of peer

We only showed makers account age before also at trade screens.
This PR fixes that so that the trade peers account age is displayed at
the peer info icon.

ManfredKarrer committed:
Merge pull request #2736 from ManfredKarrer/fix-dao-full-mode-settings-issue

Fix issues with DAO full mode in preferences

ManfredKarrer committed:
Move method

ManfredKarrer committed:
Add account age to trade details window

ManfredKarrer committed:
Add account age to contract window

ManfredKarrer committed:
Add account age to contract json

ManfredKarrer committed:
Add account age to dispute table

ManfredKarrer committed:
Hide age at altcoins

sqrrm committed:
Count cycles correctly, fix #2706

sqrrm committed:
Use same precision for all x-axis labels

ManfredKarrer committed:
Merge pull request #2737 from ManfredKarrer/use-utf8-for-getBytes

Add UTF8 for getBytes calls

ManfredKarrer committed:
Merge branch 'master' into fix-trade-peers-account-age

ManfredKarrer committed:
Dont disable the confirm buttons

Arbitrators reported that there are some cases where the confirm
payment or receipt buttons are disabled and they cannot confirm it.
Another issue is that sometimes the confirm msg does not arrive and
after button click its disabled so the message cannot be resent but
clicking the button again.
The software should handle such case and there is some automatic resend
but it seems it does not cover all cases. By keeping the button active
the user can click again and there is higher chance that such cases
get resolved by itself.
I am aware that this is not a good solution but to find out what
exactly caused the issue and to build a more resilient resend
functionality requires much more work and this seems to be a easy fix
which might avoid many disputes.

ManfredKarrer committed:
Remove check if trade is disputed

It can be that one user opened a dispute but in the meantime the other
peer sent or received the funds. Atm after a dispute has been opened the
user cannot confirm anymore. This restriction forces all cases to be
resolved in arbitration. If we relax that restriction to allow users
to still confirm after they opened a dispute we might reduce work load
for arbitrators as the payout can be done by the users if both have
confirmed. The arbitrator still need to close the case to that the
open support cases get closed, but the payout tx was already created
 by the users.

ManfredKarrer committed:
Merge pull request #2757 from sqrrm/count-voting-cycles

Count cycles correctly, fix #2706

sqrrm committed:
Refactor variable names

devinbileck committed:
Reflect pricenode operator change in ProvidersRepository

I have taken over one of the pricenodes from Manfred.

See bisq-network/roles#14

devinbileck committed:
Update mobile notification relay node DEV_URL IP address

I have taken over the relay node from Manfred. The onion address
remains the same, but the IP address used in the DEV_URL has
changed.

See bisq-network/roles#82

ManfredKarrer committed:
Merge pull request #2764 from ManfredKarrer/dont-disable-confirm-buttons

Dont disable confirm buttons

ripcurlx committed:
Merge pull request #2763 from sqrrm/market-view-precision

Use same precision for all x-axis labels

ripcurlx committed:
Merge pull request #2753 from ManfredKarrer/fix-trade-peers-account-age

Fix account age display of peer

trustfarm-dev committed:
List Trust Eth reOrigin (TEO)
Author: trustfarm.info@gmail.com
Date: Thu April 24 23:00 GMT+9

List Trust Eth reOrigin (TEO)
Official project URL: https://forum.tao.foundation
Official block explorer URL: https://explorer.tao.foundation ,
https://explorer2.tao.foundation
Official git repository URL: https://github.com/tao-foundation

ripcurlx committed:
Merge pull request #2775 from tao-foundation/list-teo-assets

List Trust Eth reOrigin (TEO)

ripcurlx committed:
Merge pull request #2770 from devinbileck/update-relay-node-address

Update mobile notification relay node DEV_URL IP address

ripcurlx committed:
Merge pull request #2769 from devinbileck/change-pricenode-operator

Reflect pricenode operator change in ProvidersRepository

ripcurlx committed:
Merge pull request #2675 from m4hdi1995/list-parsicoin

List ParsiCoin (PARS)

ripcurlx committed:
Add prompt text for filter textfield

ManfredKarrer committed:
Merge pull request #2752 from ManfredKarrer/add-filter-to-trade-history

Add filter to trade history

ManfredKarrer committed:
Add missing import

oscarguindzberg committed:
Improve logging

ripcurlx committed:
Update translations

ripcurlx committed:
Update dbs

ManfredKarrer committed:
Add check for account age to apply restrictions

ManfredKarrer committed:
Revert changes with filtering not mature accounts

- We will allow small amounts so we keep all accounts and add the check
later when the user click next

ManfredKarrer committed:
Update text

ManfredKarrer committed:
Remove check for makerHasNoMatureAccountForBuyOffer

ManfredKarrer committed:
Refactor restriction handling

ManfredKarrer committed:
Fix verify task

ManfredKarrer committed:
Add amount limit to take offer view

ManfredKarrer committed:
Improve dispute handling of failed trades

ManfredKarrer committed:
Update text

ManfredKarrer committed:
Fix limitations

ManfredKarrer committed:
Improve handling of filter

ManfredKarrer committed:
Refactoring: Rename CheckIfPeerIsBanned to ApplyFilter

ManfredKarrer committed:
Refactoring: Extract variable

ManfredKarrer committed:
Add null check

ripcurlx committed:
Only show information popup when actual amount was set

ripcurlx committed:
Merge pull request #2801 from ManfredKarrer/restrict-new-accounts

Add check for account age to apply restrictions

ripcurlx committed:
Bump version number

ripcurlx committed:
Revert to SNAPSHOT version

devinbileck committed:
Reflect mainnet seednode operator change

I have taken over one of the seednodes from @manfredkarrer.

ManfredKarrer committed:
Merge pull request #2803 from devinbileck/change-seednode-operator

Reflect mainnet seednode operator change

KrzysiekJ committed:
List burnt BlackCoin (BLK-BURNT)

ManfredKarrer committed:
Fix bug that sellers are restricted as well

Only buyers should be restricted if their account age is too young.

ManfredKarrer committed:
Reduce SAFE_ACCOUNT_AGE_DATE from 15.3. to 1.3.

We have one very suspicious case where the trader created his
account on 15. of March. To be more safe to exclude potential scammers
we decrease the check to 1st of march.

ripcurlx committed:
Improve layout of Zelle form for buyer

ripcurlx committed:
Merge pull request #2813 from ManfredKarrer/allow-seller-to-take-any-offers

Allow seller to take any offers

ripcurlx committed:
Update translations and change trusted account age from 15th of March to 1st of March

ripcurlx committed:
Bump version number

ripcurlx committed:
Revert to SNAPSHOT version

ripcurlx committed:
Fix broken SNAPSHOT renaming

ManfredKarrer committed:
Update seed node operators

ManfredKarrer committed:
Fix issue with too early disconnect to seeds

As soon we are over our connection target we start disconnecting seed
nodes though that can be too early as getBlocks requests are still
pending. We delay the disconnect seed handling to 3 minutes after our
first connection so give it enough time to receive the getBlocksResponse.

ManfredKarrer committed:
Call setAllowDisconnectSeedNodes from LiteNode after blocks received

Instead of the 3 minutes delay we set the flag from the lite node once
the blocks are received. We delay 20 seconds to allow multiple getBlocksRequests
to finish.

ManfredKarrer committed:
Cleanups

ManfredKarrer committed:
Merge pull request #2821 from ManfredKarrer/fix-connection-issue

Fix getBlocks issue

ManfredKarrer committed:
Update node operators

ManfredKarrer committed:
Merge branch 'master' of https://github.com/bisq-network/bisq

ManfredKarrer committed:
Fixes https://github.com/bisq-network/bisq/issues/2804 and https://github.com/bisq-network/bisq/issues/2826

ManfredKarrer committed:
Fix issue with label width, add deposits

ManfredKarrer committed:
Merge pull request #2827 from ManfredKarrer/fix-bsq-dust-output-issue

Fixes #2804 and #2826

ManfredKarrer committed:
Merge branch 'logging' of https://github.com/oscarguindzberg/bisq into oscarguindzberg-logging

# Conflicts:
#	core/src/main/java/bisq/core/arbitration/DisputeManager.java

ManfredKarrer committed:
Merge branch 'master' of https://github.com/bisq-network/bisq

ManfredKarrer committed:
Merge pull request #2609 from oscarguindzberg/fix-timeout

Bitcoin p2p: Use different timeouts

ManfredKarrer committed:
Merge pull request #2618 from oscarguindzberg/bitcoinj-conn-fixes

Use bitcoinj with connection handling audit fixes + Transaction.unCache() set confidence to null

ManfredKarrer commented:
@blabno I fear that PR is too far away from master. Sorry for the delay due all the circumstances, but probably better to make a new PR once devs have resources for a profound review and test.

ManfredKarrer committed:
Merge pull request #2830 from ManfredKarrer/improve-dispute-ui

Fix issue with label width, add deposits

devinbileck committed:
Fix DAO synchronization text after restore from seed

After restoring from seed, the text shown under DAO /
BSQ Wallet / Transactions displays an incorrect progress - the numbers
are swapped. For example:
"Awaiting blocks... Verified 575,868 blocks out of 554,857"

Normally we get the latest block height from BitcoinJ as the
target height, and we request BSQ blocks from seed nodes up to latest
block.

But when restoring from seed, we receive the latest block height
from the seed nodes while BitcoinJ has not received all blocks yet and
is still syncing.

Fixes https://github.com/bisq-network/bisq/issues/2825

devinbileck committed:
Update status text when BTC sync in progress

Changes the status text when BTC sync in progress from:
Synchronized with Bitcoin Mainnet (using Tor): 10.00%

to:
Synchronizing with Bitcoin Mainnet (using Tor): 10.00%

ManfredKarrer committed:
Merge pull request #2836 from devinbileck/fix-dao-sync-text-after-restore-from-seed

Fix dao sync text after restore from seed

ManfredKarrer committed:
Merge pull request #2705 from devinbileck/proposal-url-validation

Add specific URL validation for proposal links

devinbileck committed:
Fix formatDurationAsWords when duration is multiple of 10 days

When the duration is a multiple of 10 days (e.g. 110 days) and not
showing zero values, it was incorrectly formatting it as "11" rather
than "110 days".

devinbileck committed:
Add more tests for formatDurationAsWords

sqrrm committed:
Merge pull request #2844 from devinbileck/fix-formatDurationAsWords

Fix confirm lockup transaction prompt showing incorrect estimated time

sqrrm committed:
Use minTakerFee when calculating taker fee

sqrrm committed:
Merge pull request #2846 from sqrrm/use-correct-minfee

Use minTakerFee when calculating taker fee

ripcurlx committed:
Merge pull request #2750 from zzzpotato/list-zeroclassic-asset

List ZeroClassic (ZERC)

sqrrm committed:
Show peer address properly

sqrrm committed:
Handle encrypted BSQ wallet

ripcurlx committed:
Merge pull request #2811 from KrzysiekJ/list-burnt-blackcoin

List burnt BlackCoin (BLK-BURNT)

ripcurlx committed:
Merge pull request #2854 from sqrrm/fix-peer-handling

Fix peer handling

y3v63n committed:
Update displayStrings.properties

Fixing linguistic issues in the English version

sqrrm committed:
Merge pull request #2858 from y3v63n/master

Update displayStrings.properties

agb19 committed:
Fix "useless use of IPv6" in Tor socket

Implements the "possible fix" described in issue #2840 by appending the directive `java.net.preferIPv4Stack=true` to the JVM options. 
Tested successfully on Debian 9 and Tails 3.14, using platform Tor and onion-grater.

sqrrm committed:
Merge pull request #2862 from agb19/patch-1

Fix "useless use of IPv6" in Tor socket

Emzy committed:
node1.emzy.de has moved to a new server

New IP address for node1.emzy.de is 167.86.90.239

ripcurlx committed:
Merge pull request #2864 from Emzy/patch-1

node1.emzy.de has moved to a new server

ripcurlx committed:
Update translations

ripcurlx committed:
Update stores

ripcurlx committed:
Bump version number

ripcurlx committed:
Revert to SNAPSHOT version

freimair committed:
Fix NullPointerEx in Monitor

freimair committed:
Gather Dao data

freimair committed:
SeedNodeSnapshot knows the Dao

freimair committed:
Increase max message size until a greater refactoring

freimair committed:
Removed static bsq block height

freimair committed:
Renamed classes

ripcurlx committed:
Merge pull request #2870 from freimair/monitor-bsqblocks

Monitor knows about the DAO

Commit:
Minimal HTTP API

Commit:
Decrease logs verbosity for integration tests

Commit:
Security framework for the API

Commit:
API over TOR

Commit:
Payment account support for the API
